packages/search-ui-elasticsearch-connector/src/handlers/search/Configuration.ts

```diff
@@ -63,7 +63,8 @@ function buildConfiguration(
             field: facetKey,
             label: facetKey,
             size: facetConfiguration.size || 20,
-            multipleSelect: isDisJunctive
+            multipleSelect: isDisJunctive,
+            order: facetConfiguration.sort || "count"
```

Comment:
Just wondering, what do you think about defaulting to alphabetical?

Reply:
yeh I went with count as thats the default behaviour when not specified in app-search / swifttype.